flake8_html/plugin.py

```diff
@@ -13,6 +13,8 @@
 import codecs
 import datetime
 import pkgutil
+import configparser
```

Comment:
Having briefly looked into how flake8 parses options, I don't think using configparser directly is necessary at all.

[Flake8 allows us to specify options that can be parsed from config](https://gitlab.com/pycqa/flake8/blob/master/src/flake8/options/manager.py#L60). While flake8's plugin documentation isn't complete, and so it's debatable whether this is a public API, this seems more appropriate than re-inventing it ourselves.